docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -35,232 +34,188 @@ metadata are entered in the live catalogue.
 
 ![MOLGENIS databases](../img/cat_databases.png)
 
 <sup>*Figure 1. Databases in the Data Catalogue staging area.*</sup>
 
 ### Fill out cohort rich metadata
 
-Open your staging area, navigate to 'Tables' and open the table 'Cohorts'. Your cohort pid and name are already 
+Open your staging area, navigate to 'Tables' and open the table 'Cohorts'. Your cohort id and name are already 
 filled out. Click on the pencil sign next to this entry to start editing your cohort rich metadata by filling out 
-the form.
+the form. 'Subcohorts' and 'Collection events' should also be filled out through this route. You can fill them out 
+in subsections inside the 'Cohorts' form.
 
 ### Define metadata of cohorts or data sources
 
 This section explains how to submit the 'source variables' + 'mappings from source variables to target variables' into
 the Data Catalogue. Expected users of this 'how to' are data managers within the institutions with access to cohorts or
 data sources. This document assumes you have received login details for upload of your metadata. You can also watch
-this [*instruction video*](https://www.youtube.com/watch?v=b_Ef_Uiw1gE&amp;ab_channel=MOLGENIS).
+this [*instruction video*](https://www.youtube.com/watch?v=b_Ef_Uiw1gE&amp;ab_channel=MOLGENIS). Note that this video used the old dictionary model.
 
 #### Define source variable metadata / source data dictionary
 
-We use the [*Dictionary template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceDictionary.xlsx)
+We use the [*SourceDictionary template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceDictionary.xlsx)
 to define variable metadata. The [*SourceDictionary
 template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceDictionary.xlsx) consists of multiple
 sheets. Each sheet corresponds to a table in the Data Catalogue (Figure 1). The columns in the sheet correspond to
 columns in the table concerned. This document describes how to fill out each of the sheets and their columns. A column
 with an asterisk (\*) after its name is mandatory, i.e., it should contain values for the system to accept a data
 upload. You can download this 
 [*filled out example*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/SourceDictionary_testCohort.xlsx) 
-as a reference for filling out the template.
-
-<sup>Note that there is no sheet for *AllSourceVariables*. This table is a generic listing of all variables entered for
-the cohort; it shows *SourceVariables* and *RepeatedSourceVariables* in one table. </sup>
+as a reference for filling out the template. Note that there is no sheet for *AllVariables*. This table is a generic listing of all variables entered for
+the cohort; it shows *Variables* and *RepeatedVariables* in one table.
 
 It is good practice to try adding a few variables to the template first and see whether your upload succeeds. To
```

Comment:
might need to add additional troubleshooting page if uploading of variables fails

Reply:
out of scope for this PR and story